lib/rules/string-quotes/__tests__/index.js

```diff
@@ -168,6 +184,64 @@ testRule(rule, {
 
 testRule(rule, {
   ruleName,
+  config: ["single", { avoidEscape: false }],
+  skipBasicChecks: true,
+  reject: [
```

Comment:
Please add `fix: true` for all new tests.

Reply:
After some investigation and thinking, I withdraw my comment. `testRule()` designed the way it is on purpose. If a rule has autofix and if a code has a violation, then fixed code should be different than input code. It also prevents us from human error, but copy-pasting `code` to `fixed` and forgetting about it.

Please add a comment before these two `testRule` why there no `fix` set (fix grammar or rephrase if needed):

```js
// Styles which requires escaping isn't autofixed, it's just reported instead
```